tests/schema.py

```diff
@@ -258,7 +258,7 @@ class SimpleSource(dj.Lookup):
     definition = """
     id : int  # id
     """
-    contents = ((x,) for x in range(10))
+    contents = [(x,) for x in range(10)]
```

Comment:
Similar to https://github.com/datajoint/datajoint-python/pull/1129#discussion_r1424706196, this generator was exhausted once, causing failures for additional tests that required non-empty `SimpleSource.contents`.